samples/bluetooth/central_hids/sample.yaml

```diff
@@ -19,9 +21,12 @@ tests:
       - nrf5340dk/nrf5340/cpuapp
       - nrf5340dk/nrf5340/cpuapp/ns
       - nrf54l15dk/nrf54l15/cpuapp
-      - nrf54l15pdk/nrf54l15/cpuapp
       - nrf54h20dk/nrf54h20/cpuapp
-    platform_allow: nrf52dk/nrf52832 nrf52840dk/nrf52840 nrf5340dk/nrf5340/cpuapp
-      nrf5340dk/nrf5340/cpuapp/ns nrf54l15pdk/nrf54l15/cpuapp nrf54l15dk/nrf54l15/cpuapp
-      nrf54h20dk/nrf54h20/cpuapp
+    platform_allow:
+      - nrf52dk/nrf52832
```

Comment:
Btw. Why do we keep boards from `sample.bluetooth.central_hids` also under this `build_only` setup? Unless I missed something, the duplication should not be needed.

Reply:
I also wonder how the `harness: bluetooth` works. Looking at Twister docs (https://docs.zephyrproject.org/latest/develop/test/twister.html) it seems that it does nothing (unless our integration CIs make a custom use of the harness).

Maybe we could consider removing it and merging both test scenarios? (maybe we could even define a simple harness config to verify if sample boots properly)

Reply:
Similar comments apply to other modified samples

Reply:
Discussed with @kapi-no. Prepared follow-up tickets: `NCSDK-30419`, `NCSDK-30420`